doc/index.md

```diff
@@ -32,3 +32,62 @@ In PostgreSQL, this implicitly uses `template1` as the template - anything exist
 be copied to your new database. If you wish to change the database used as a template, you can specify
 the `EF Template Database` connection string parameter. For more info see the
 [PostgreSQL docs](https://www.postgresql.org/docs/current/static/sql-createdatabase.html).
+
+## Customizing DataReader Behavior ##
+
+You can use [an Entity Framework 6 IDbCommandInterceptor](https://msdn.microsoft.com/en-us/library/dn469464(v=vs.113).aspx) to wrap the ```DataReader``` instance returned by Npgsql when Entity Framework executes queries. This is possible using a ```DbConfiguration``` class.
```

Comment:
Of course github markdown screwed it up. I just meant to use one backtick and not three around DataReader...